src/js/state.js

```diff
@@ -41,6 +41,14 @@ export default async function () {
   } else {
     ciphers = ciphers;
   }
+  if (form['config'].value === 'old') {
```

Comment:
Could that be inferred from `protocols` perhaps at this point already, instead of hardcoding it? (in case the profiles change over time…)

Reply:
I think it would be more fragile for the code to infer it, since then this code may need to be modified when the ssl-config-generator spec is (eventually) updated.  By contrast, if we were to rename profiles, that is easier to identify profile name use in the code.  I came across the config name use in caddy.hbs earlier today, so there is at least that also referencing the profile name.

Reply:
Quite the opposite, once the protocols fall off the cliff, this will stop lowering the seclevel for us (I'd prefer to test on features… profile names should only be used for decorative purposes, they don't bear any logic per se… caddy example is one of the worst sins;D)

Or you mean there are other algos that may need the 80–112bit range even once the weak protocols are out? That's a fair point, but TBH something we can't assess right now, when we don't know the next "old" specs, and what feature might need the seclevel tuning (and then, I'd still like to test for such feature if possible, instead of the whole profile… let's see what the next json update brings.)

Reply:
yes, testing for feature in updates to this PR